staging/operator-lifecycle-manager/pkg/controller/registry/reconciler/grpc_test.go

```diff
@@ -450,15 +459,19 @@ func TestRegistryPodPriorityClass(t *testing.T) {
 			stopc := make(chan struct{})
 			defer close(stopc)
 
-			factory, client := fakeReconcilerFactory(t, stopc, withNow(now), withK8sObjs(tt.in.cluster.k8sObjs...), withK8sClientOptions(clientfake.WithNameGeneration(t)))
+			// a defaultNamespace resource must be present so that the reconciler can determine the
+			// security context configuration for the underlying pod
+			clusterState := append(tt.in.cluster.k8sObjs, defaultNamespace())
```

Comment:
Think this uses the new func and is a bit easier to read and reason about:

```suggestion
			clusterState := baseClusterState()```